docker-compose.yaml

```diff
@@ -5,7 +5,4 @@ services:
     image: ghcr.io/energywebfoundation/ewx-worker-node-server/ewx-worker-node-server:latest
     ports:
       - '3002:3002'
-    environment:
```

Comment:
Lets leave this one for now  as it was @hejkerooo , as it is aligned with instruction on readme

Reply:
Ok, maybe lets do it like that:

lets modify here (https://github.com/energywebfoundation/ewx-worker-node-server-fork?tab=readme-ov-file#using-docker-compose) , so it says "Update VOTING_WORKER_SEED in .env.default and then run....

And in docker compose we can just point to that env example file